backend/src/main/java/com/team4/backend/service/UserService.java

```diff
@@ -30,9 +31,14 @@ public UserService(UserRepository userRepository, PBKDF2Encoder pbkdf2Encoder, J
         return userRepository.findByEmailAndPasswordAndIsEnabledTrue(authRequestDto.getEmail(), pbkdf2Encoder.encode(authRequestDto.getPassword()))
                 .map(jwtUtil::generateToken)
                 .switchIfEmpty(Mono.error(new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Can't find user with this credentials")));
+    }
 
+    public Mono<User> findByEmail(String email) {
```

Comment:
do we need this function (can we use the isEmailTaken method instead)

Reply:
![image](https://user-images.githubusercontent.com/49820008/134608329-6a59761f-56e8-4f3b-8bc7-50eead20d8cf.png)

Reply:
Was more of a personal note